modules/meshrenderinggl/src/rendering/fragmentlistrenderer.cpp

```diff
@@ -146,9 +146,6 @@ bool FragmentListRenderer::postPass(bool useIllustration, const Image* backgroun
     // check if enough space was available
     if (numFrags > fragmentSize_) {
         // we have to resize the fragment storage buffer
-        LogInfo("fragment lists resolved, pixels drawn: "

```

Comment:
maybe we could expose the memory usage as a readonly property or something? other PR probably...

Reply:
Maybe. On the other hand I never experienced any memory overflow on a 4K screen, so hopefully the error will never be a problem.

Reply:
When i rewrote that code I did some tests, and as long as you only use one, renderer it not to bad. but If you have a couple of renderers and use illustration buffers, then you easily use several GBs of memory...